grails-app/services/org/pih/warehouse/core/LocationService.groovy

```diff
@@ -18,17 +18,16 @@ import org.apache.poi.hssf.usermodel.HSSFWorkbook
 import org.apache.poi.ss.usermodel.Cell
 import org.apache.poi.ss.usermodel.Row
 import grails.plugins.csv.CSVMapReader
+import org.pih.warehouse.data.LocationDataService
 import org.pih.warehouse.importer.ImportDataCommand
 import util.ConfigHelper
 
 @Transactional
 class LocationService {
 
     GrailsApplication grailsApplication
-    def dataService
-    def productTypeService
-    def userService
-
```

Comment:
how about those? are they used? You mentioned only `dataService` that is not used

Reply:
@kchelstowski @jmiranda 
userService:
![image](https://github.com/openboxes/openboxes/assets/83239466/c3e31302-26ae-4892-8742-eaa4cf9e4136)
locationDataService:
![image](https://github.com/openboxes/openboxes/assets/83239466/2ecf1a84-3a9c-4b91-b21f-6d3a5a0e5096)